lib/Star/Kata/Builder/KataBuilder.php

```diff
@@ -0,0 +1,99 @@
+<?php
+/**
+ * This file is part of the phpkata project.
+ *
+ * (c) Yannick Voyer (http://github.com/yvoyer)
+ */
+
+namespace Star\Kata\Builder;
+
+use Star\Kata\Model\Kata;
+use Star\Kata\Model\Objective\Objective;
+
+/**
+ * Class KataBuilder
+ *
+ * @author  Yannick Voyer (http://github.com/yvoyer)
+ *
+ * @package Star\Kata\Builder
+ */
+final class KataBuilder
+{
+    /**
+     * @var ObjectiveBuilder
+     */
+    private $objectiveBuilder;
+
+    /**
+     * @var array
+     */
+    private $data;
+
+    /**
+     * @var Objective[]
+     */
+    private $objectives = array();
+
+    /**
+     * @var int
+     */
+    private $count = 1;
+
+    public function __construct()
+    {
+        $this->resetData();
+        $this->objectiveBuilder = new ObjectiveBuilder();
+    }
+
+    /**
+     * @return Kata
+     */
+    public function build()
+    {
+        $this->count ++;
```

Comment:
`count` is not a very good variable name.. what is it counting and what purpose does it serve?